qa/automation/CareerPortal.Tests.Scenarios/5. Hyperlinks/5.1. Hyperlinks on the page.feature

```diff
@@ -0,0 +1,51 @@
+Feature: 5.1 hyperlinks on the page
+Background:
+	Given User opened Career Portal
+	And user hovers over a hyperlink
+
+
+Scenario: 5.1_01 Change cursor to pointer
+	When cursor is moved to the "Quantori" hyperlink area
+	Then the cursor is changed to pointer
+
+Scenario: 5.1_02 Change cursor to pointer
+	When cursor is moved to the "About" hyperlink area
+	Then the cursor is changed to pointer
+
+Scenario: 5.1_03 Change cursor to pointer
+	When cursor is moved to the "Open job positions" hyperlink area
+	Then the cursor is changed to pointer
+
+Scenario: 5.1_04 Change cursor to pointer
+	When cursor is moved to the "Contact" hyperlink area
+	Then the cursor is changed to pointer
+
+Scenario: 5.1_05 Change cursor to pointer
+	When cursor is moved to the "Data Privacy Policy" hyperlink area
+	Then the cursor is changed to pointer
+
+Scenario: 5.1_06 Change cursor to pointer
+	When cursor is moved to the "Cookies Settings" hyperlink area
+	Then the cursor is changed to pointer
+
```

Comment:
5.1_01 - 5.1_06 can be combined to one Scenario Outline as cases for buttons above